src/boss/cloudvm/local_platform_config.go

```diff
@@ -4,13 +4,19 @@ import (
 	"log"
 	"os"
 	"path/filepath"
+	"strconv"
 )
 
 type LocalPlatConfig struct {
 	Worker_Starting_Port           string `json:"worker_starting_port"`
 	Path_To_Worker_Config_Template string `json:"path_to_worker_config_template"`
 }
 
+var (
```

Comment:
currentWorkerPort (actually "nextWorkerPort" is probably a better name) would make more sense in our struct (https://github.com/open-lambda/open-lambda/blob/main/src/boss/cloudvm/local_worker.go#L14) rather than as a global variable.  You can get rid of portInitialized while you're at it.